calculator.rb

```diff
@@ -0,0 +1,111 @@
+# If the result is a whole number, program should return an integer. Otherwise, it should return a float rounded to 2 decimal points.
+def checkForDecimal(result)
+  if result % 1 == 0
+    result = result.to_i
+  else
+    result = result.to_f.round(2)
+  end
+end
+
+# Methods for each operator
+def add(num1, num2)
+  result = num1 + num2
+  checkForDecimal(result)
```

Comment:
You call `checkForDecimal` in each of these methods - is there somewhere else that you could put this so you only have to write it once?

Reply:
I looked through it with one of the tutors, and we couldn't think a way to write it only once, but I was able to get it down to two times. I moved checkForDecimal and put it under getUserNumber in lines 61 and 65 on my newly committed code.